packs/bit-bucket/requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.6.0
```

Comment:
I agree with @jfryman.

In addition to that, upstream library already implements oauth support - Implementing that ourselves would be time consuming and a waste of time so we should just use the upstream library.